drivers/hmis/app/models/hmis/hud/hmis_service.rb

```diff
@@ -16,9 +16,43 @@ class Hmis::Hud::HmisService < Hmis::Hud::Base
 
   belongs_to :owner, polymorphic: true # Service or CustomService
   belongs_to :custom_service_type
+  has_many :custom_service_categories, through: :custom_service_type
+
   alias_attribute :service_type, :custom_service_type
+  alias_to_underscore [:DateProvided]
 
   SORT_OPTIONS = [:date_provided].freeze
+  HUD_ATTRIBUTES = [:record_type, :type_provided, :other_type_provided, :moving_on_other_type, :sub_type_provided, :referral_outcome, :FAAmount].freeze
```

Comment:
What's the reason behind the capitalization of `:FAAmount`?  Is that just because it's an odd case?

Reply:
Yeah, that's a pattern we have in the graphql types. We use the original casing for weirdly cased attributes (FAAmount, HMISParticipatingProject, HOPWAMedAssistedLivingFac, etc). I'm not really sure why honestly. I can look into see whether it's really necessary. It may be to avoid having something like h_m_i_s_

Reply:
I think we can standardize them more, but not sure if its worth spending time on. If we do it at all it will be a separate PR because it results in several schema changes.